src/DssSpell.sol

```diff
@@ -13,85 +13,57 @@
 //
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
-pragma solidity 0.6.11;
+pragma solidity 0.6.12;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
+// https://github.com/makerdao/ilk-registry/blob/master/src/IlkRegistry.sol
+interface IlkRegistryLike {
+    function list() external view returns (bytes32[] memory);
+}
+
 contract DssSpellAction is DssAction {
 
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/a083a4efccd26e23b7d5493f019fa5999997c1e5/governance/votes/Executive%20vote%20-%20March%2012%2C%202021.md -q -O - 2>/dev/null)"
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/e0a37c7b58a98566ba637793a804179b4666b4c7/governance/votes/Executive%20vote%20-%20March%2019%2C%202021.md -q -O - 2>/dev/null)"
     string public constant description =
-        "2021-03-12 MakerDAO Executive Spell | Hash: 0x100830e8d607164a38aeff906e85ed79c7007fd92bac416ff7f2fc36935eff8e";
+        "2021-03-19 MakerDAO Executive Spell | Hash: 0xa7979ce287cd7df9ecbf25251f2b41a52c27a47567721d7c5ddfc6f85885d4d7";
+
+    uint256 constant MILLION = 10**6;
 
+    // Disable Office Hours 
+    function officeHours() public override returns (bool) {
+        return false;
+    }
 
-    // Many of the settings that change weekly rely on the rate accumulator
-    // described at https://docs.makerdao.com/smart-contract-modules/rates-module
-    // To check this yourself, use the following rate calculation (example 8%):
-    //
-    // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
-    //
-    // A table of rates can be found at
-    //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
-    //
-    uint256 constant TWO_PCT            = 1000000000627937192491029810;
-    uint256 constant THREE_PCT          = 1000000000937303470807876289;
-    uint256 constant THREE_PT_FIVE_PCT  = 1000000001090862085746321732;
-    uint256 constant FOUR_PCT           = 1000000001243680656318820312;
-    uint256 constant FOUR_PT_FIVE_PCT   = 1000000001395766281313196627;
-    uint256 constant FIVE_PCT           = 1000000001547125957863212448;
+    function actions() public override {
 
-    uint256 constant MILLION    = 10**6;
-    uint256 constant WAD        = 10**18;
-    uint256 constant RAD        = 10**45;
+        address ILK_REGISTRY = DssExecLib.getChangelogAddress("ILK_REGISTRY");
+        IlkRegistryLike registry = IlkRegistryLike(ILK_REGISTRY);
```

Comment:
You should have access to this address using `DssExecLib.reg()`, you could probably collapse this down to one line with:

```
bytes32[] memory ilks = IlkRegistryLike(DssExecLib.reg()).list();
```